### ISSUE TITLE
schema-reader-mg Error: Failed opening required 'C:\laragon\www\vemtoTest\bootstrap/../routes/app.php' (include_path='.;C:/laragon/etc/php/pear')

### ISSUE THREAD

**Issue by Flat-Duck** [VemtoOrg/vemto2-issues#120]
**Error Message:**
schema-reader-mg Error: Failed opening required 'C:\laragon\www\vemtoTest\bootstrap/../routes/app.php' (include_path='.;C:/laragon/etc/php/pear')

**Stack:**
```
schema-reader-mg Error: #0 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Routing\Router.php(511): Illuminate\Routing\RouteFileRegistrar->register('C:\\laragon\\www\\...')
#1 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Routing\Router.php(465): Illuminate\Routing\Router->loadRoutes('C:\\laragon\\www\\...')
#2 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Routing\RouteRegistrar.php(194): Illuminate\Routing\Router->group(Array, 'C:\\laragon\\www\\...')
#3 C:\laragon\www\vemtoTest\bootstrap\app.php(18): Illuminate\Routing\RouteRegistrar->group('C:\\laragon\\www\\...')
#4 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Configuration\ApplicationBuilder.php(216): {closure}(Object(Illuminate\Foundation\Application))
#5 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Illuminate\Foundation\Configuration\ApplicationBuilder->Illuminate\Foundation\Configuration\{closure}()
#6 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\Util.php(41): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#7 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(81): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#8 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Object(Closure), Object(Closure))
#9 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\Container.php(662): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Object(Closure), Array, NULL)
#10 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(138): Illuminate\Container\Container->call(Object(Closure))
#11 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(52): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Illuminate\Foundation\Support\Providers\RouteServiceProvider->Illuminate\Foundation\Support\Providers\{closure}()
#13 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\Util.php(41): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(81): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#15 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Object(Closure), Object(Closure))
#16 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Container\Container.php(662): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Object(Closure), Array, NULL)
#17 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Support\ServiceProvider.php(126): Illuminate\Container\Container->call(Object(Closure))
#18 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(1106): Illuminate\Support\ServiceProvider->callBootedCallbacks()
#19 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(1084): Illuminate\Foundation\Application->bootProvider(Object(Illuminate\Foundation\Support\Providers\RouteServiceProvider))
#20 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(Illuminate\Foundation\Support\Providers\RouteServiceProvider), 28)
#21 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(1083): array_walk(Array, Object(Closure))
#22 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#23 C:\laragon\www\vemtoTest\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(309): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#24 phar://C:/Users/macroMan/AppData/Local/Programs/Vemto2/resources/static/VMTTL1/index.php(37): Illuminate\Foundation\Application->bootstrapWith(Array)
#25 phar://C:/Users/macroMan/AppData/Local/Programs/Vemto2/resources/static/VMTTL1/common/Vemto.php(111): {closure}()
#26 phar://C:/Users/macroMan/AppData/Local/Programs/Vemto2/resources/static/VMTTL1/index.php(23): Vemto\Vemto::execute('schema-reader-m...', Object(Closure))
#27 C:\Users\macroMan\AppData\Local\Programs\Vemto2\resources\static\VMTTL1(14): require('phar://C:/Users...')
#28 {main}
```

---

I tried to make new project using vemto 2 it said wait to finish  after building finished it showed me a connect to app dialog i chose migration reader and selected fresh laravel app and i pressed connect it opened the ide but nothing was showing on the bottom right Red error dot flashing and read it it was the same as above problem then closed the app reopened it and it will not connect giving this error i changed it to DB reader and it won't connect bcz there is no DB in mysql server   i closed the a
Vemto 2.0.9
win32 10.0.22631
Locale: en-US
App version: 2.0.9

**Comment by Flat-Duck** [comment]
also using composer update on the terminal outside vemto on the same app seemed to cuase the same error 
![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/7350041/c41cdde7-9b26-45f4-ac03-3a9f3b05d6e7)

i think vemto doesn't copy this `bootstrap/../routes/app.php` file

**Comment by Flat-Duck** [comment]
after creating new laravel project using `laravel new` command  and connecting it with vemto and selecting fresh laravel app the same issue appeared and also in terminal it showed up 
while creating another app and connecting it to vemto without selecting fresh laravel app the problem did not happen.
from all this i can see that vemto edits the bootstrap/app.php file 

      ` 
        then: function() {
            Route::middleware('web')->group(__DIR__.'/../routes/app.php');
        }`
        adding this change results in 
`              Illuminate\Routing\RouteFileRegistrar::register("C:\laragon\www\vi\bootstrap/../routes/app.php")`
i think that __DIR__ doesn't refere to app base folder as it looks for the file inside bootstrap folder

**Comment by TiagoSilvaPereira** [comment]
Hi @Flat-Duck thank you for reporting, I fixed it on 2.0.10
